alpha_beta.pl

```diff
@@ -0,0 +1,40 @@
+% -------------------------------------------------------------------------------
+% Import heuristic
+:- [heuristics].
+% -------------------------------------------------------------------------------
+
+alphabeta(Depth, Pos, Alpha, Beta, GoodPos, Val):-
```

Comment:
Pos is BoardState? Pos for POSition? POSibble move?

Reply:
I don't understand half of the var names used in this file..
Maybe it's a generic code, but I still think they can be more meaningful